imapbackup.py

```diff
@@ -124,6 +124,20 @@ def string_from_file(value):
         return content.read().strip()
 
 
+class Narrator:
```

Comment:
Dead-code or in-direct reference?

Reply:
Oh, whoops.  That wasn't supposed to be commited.